### PR TITLE
Uar 1584 add property land owner nocs for individual

### PR DESCRIPTION
**Pull request by markpit**
### JIRA link
https://companieshouse.atlassian.net/browse/UAR-1584

### Change description
Add new natures of controls to the BO individual page on Registration journey, 'Tell us about the individual person'
Switched off with feature flag FEATURE_FLAG_ENABLE_PROPERTY_OR_LAND_OWNER_NOC_30082024

### Work checklist

- [ ] Tests added where applicable
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

**Review by lduranteau — APPROVED** [review]

**Comment by lduranteau** [comment]
There is 1 smell code due to the word TODO in the comment

File: src/controllers/shared/common.resume.submission.controller.ts [hunk 1]
```diff
@@ -85,7 +85,7 @@ export const getResumePage = async (req: Request, res: Response, next: NextFunct
  * @param overseaEntityId
  */
 const setWebApplicationData = (session: Session, appData: ApplicationData, transactionId: string, overseaEntityId: string) => {
-
+  // TODO map the 3 new NOCs once added to API and SDK - (trust control and 2 x jurisdiction nocs)
```
Comment (mwestacott) [inline src/controllers/shared/common.resume.submission.controller.ts:88]: I think this TODO comment can be removed. SDK and API work both covered by the API story (other JIRA ticket).

Comment (markpit) [reply src/controllers/shared/common.resume.submission.controller.ts]: removed the Todo

**Comment by mwestacott** [comment]
LGTM

**Review by mwestacott — APPROVED** [review]

**Pull request merged**